guides/Getting_Started_(Android).md

```diff
@@ -0,0 +1,59 @@
+Getting Started
+===
+
+This guide explains how to start using LÖVR on Android devices like the Oculus Go and Oculus Quest.
+
+After setting the device up for development, the LÖVR Android .apk is "sideloaded" onto the device.
+From there, a LÖVR project on a PC can be sync'd to the device, and will automatically reload
+whenever the files are changed.
+
+Device Setup
+---
+
+First, make sure your device is set up for development.  Oculus has official device setup guides
+for both the [Oculus Quest](https://developer.oculus.com/documentation/quest/latest/concepts/mobile-device-setup-quest/)
+and [Oculus Go](https://developer.oculus.com/documentation/mobilesdk/latest/concepts/mobile-device-setup-go/),
+but there are lots of other guides on the internet for how to do this.  The key things are:
+
+- Enabling development mode on the device.
+- Installing the `adb` tool used to communicate with the device.
+
+Install the APK
+---
+
+Download the latest release version of the appsample apk from the [`lovr-oculus-mobile` release
+page](https://github.com/mcclure/lovr-oculus-mobile/releases).
+
+Install it to the device:
+
+```
+adb install /path/to/appsample-release.apk
+```
+
+Try running it by navigating to the "Library" -> "Unknown Sources" menu of the headset and running
+the `org.lovr.test` app.  You should see LÖVR's "no game" screen with the logo.
+
+Running a Project
+---
+
+Now we can create a LÖVR project, which is a folder with some code and assets in it.  Create a
+folder called `hello-world` and add this code to a file named `main.lua` in there:
+
+```
+function lovr.draw()
+  lovr.graphics.print('hello world', 0, 1.7, -3, .5)
+end
+```
+
+Then use `adb` to sync it to the device (this can be done while the app is running):
+
+```
+adb push --sync /path/to/hello-world /sdcard/Android/data/org.lovr.test/files/.lodr
```

Comment:
Two small notes.'
* this adb line will work. you have to do

        cd /path/to/hello-world && adb push --sync . /sdcard/Android/data/org.lovr.test/files/.lodr

    I think this also works:

            adb push --sync /path/to/hello-world/. /sdcard/Android/data/org.lovr.test/files/.lodr

    But I don't remember exactly. I just remember there's some **incredibly** annoying rule concerning it either being required to push `.` or the path having to end in `.` and `cd`ing being the easiest way to explain things.

* The `.lodr` component was something added in 0.13, and before 0.13 that last path component was not present. If this guide goes up at the same time as 0.13 then this is no problem.